src/version_updater/version.hpp.in

```diff
@@ -12,4 +12,6 @@ char const * const smVersionFull = "@SM_VERSION_FULL@";
 #endif
 char const * const smVersionTraditional = "@SM_VERSION_TRADITIONAL@";
 
+extern char const * const smVersionDate = "@SM_TIMESTAMP_DATE@";
```

Comment:
Is `version.hpp.in` actually being used anymore? If not, I would advise removing it.